HIP/hip-583.md

```diff
@@ -10,220 +10,452 @@ status: Accepted
+is effectively locked until auto-renew fees drain the account or somebody is able to produce the key. If somebody does
+have the keys to that account, they can access the account and control it. Since Ethereum-native wallets do not
+understand the `0.0.10` style format of a Hedera account ID, it is necessary to support the creation of accounts with an
+`evm address alias` to enable Ethereum-native wallet workflows for auto-creation of accounts.
+
+### Smart Contract Integration
+
+Every account is known to the EVM by a 20-byte address. That address *may* be the `account-num alias` or the
+`evm address alias` depending on how the account was created and what type of `alias`, if any, is on the account.
+
+| Account Key Type | Account Alias               | Address Known to the EVM    |
+|------------------|-----------------------------|-----------------------------|
+| ANY              | None                        | `account-num alias`         |
+| ANY              | ECDSA secp256k1 `key alias` | `ECDSA` derived evm address |
+| ANY              | `evm address alias`         | `evm address alias`         |
+
+### Update Account with Alias
+
+[HIP 32](https://hips.hedera.com/hip/hip-32) specifies:
+> An account created normally has no alias. It can be given an alias with an account update, but only if that alias is
+> not currently used by any other account, and only if the update transaction is signed by the private key corresponding
+> to the public key in the alias.
+
+This HIP **SUPERSEDES** `HIP-32` by removing this capability. Support for updating an account with an `alias` introduces
+significant complication. If an account without an alias has already interacted with a smart contract, then it is known
+to the smart contract with the "long-zero" `account-num alias` that all accounts have. If that account were to be
+updated with an alias, then which evm address should be used in the EVM? For compatibility, it should use the
+`account-num alias`, at least with contracts it had already interacted with. However, this may run contrary to some
+expectations. While this is an important user flow we want to support, we defer this support to a future HIP. A full
+specification for this capability will likely require the ability for the user to select which alias is their default,
+and to specify on a per-call basis the alias they want to use when interacting with a smart contract.
+
+User stories
+------------------------------------------------------------------------------------------------------------------------
+
+1. As a user with an `ECDSA` derived account from another chain who uses an Ethereum-compatible wallet, I would like to
+   create my Hedera account using that same ECDSA public key and associated Ethereum compatible `evm address alias`.
+2. As a developer, I would like to create a new account with a `key alias` via the `CryptoCreate` transaction.
```

Comment:
Is this actually something we want? Should we deprecate support for `key alias` instead, and not dig the hole any deeper?

Reply:
Are we debating to drop almost all CryptoCreate changes with the exception of CryptoCreate with EVM address and provided key?

Reply:
This was explicitly requested by wallets as they value the ability to create zero balance accounts. They can do today via public key but want to extend business practice to include evm address